django_hana/compiler.py

```diff
@@ -31,6 +33,12 @@ def as_sql(self):
         return result, update_params
 
 class SQLInsertCompiler(compiler.SQLInsertCompiler, SQLCompiler):
+    def __createPlaceholder(self, field, val):
```

Comment:
Can you create a `compat.py` similar to the approach of django rest framework? https://github.com/tomchristie/django-rest-framework/blob/master/CONTRIBUTING.md#managing-compatibility-issues